src/forEachChild.ts

```diff
@@ -107,8 +107,8 @@ export function forEachChild(node: any, visitor: (node: SyntaxKind) => SyntaxKin
     case SyntaxKind.ArrowFunction:
       return (
         visitNode(node.arrowToken, node, visitor) ||
```

Comment:
can't walk a 'childLess' arrowToken node CST node. It doesn't have any children. and 'parameters' is safe to walk